cmd/deploy/deploy.go

```diff
@@ -323,8 +317,8 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 
 	if err != nil {
 		err = oktetoErrors.UserError{
-			E:    fmt.Errorf(oktetoErrors.ErrDeployHasFailedCommand, err),
-			Hint: "Please try updating your deploy section on your manifest and try again",
+			E:    oktetoErrors.ErrDeployHasFailedCommand,
+			Hint: "Update the 'deploy' section of your okteto manifest and try again",
```

Comment:
Can we give a more specific hint here? Like what the user might need to update in the `deploy` section?

Reply:
Any suggestion? Basically, the user needs to modify the `deploy` section to configure the deployment of the development environment